docs/content/getting-started.md

```diff
@@ -115,6 +115,17 @@ const app = new App({
 
 Save your `app.js` file, then on the command line run the following:
 
+Note: if you copy & pasted above code. than, you have to setup ```.env``` in your project. in order to do so follow bellow steps:
```

Comment:
You do not have to setup `.env` to take advantage of the above code. The only requirement is that the `SLACK_BOT_TOKEN` and `SLACK_SIGNING_SECRET` environment variables are exported in the current shell session, which is explicitly called out in an earlier step.

I would prefer not to complicate the getting started document and would leave additions like `.env` support as a userland concern. Application templates could adopt this (and some of our Slack-authored ones even do, for example: https://github.com/slack-samples/bolt-js-starter-template?tab=readme-ov-file#environment-variables).